python/docs/source/reference/pyspark.sql/index.rst

```diff
@@ -37,3 +37,4 @@ This page gives an overview of all public Spark SQL API.
     window
     grouping
     catalog
```

Comment:
Let's also add `unwrap_udt`. Seems like https://github.com/apache/spark/commit/daedcd29630de42f6d0858e1e693f4a3d9caf1aa missed it.